pkg/ecdsa/tss/protocol_announce.go

```diff
@@ -40,6 +41,14 @@ func AnnounceProtocol(
 
 	receivedMemberIDs := make(map[string]MemberID) // member address -> memberID
 
+	markAnnounced := func(memberID MemberID, memberAddress string) {
+		receivedMemberIDs[strings.ToLower(memberAddress)] = memberID
```

Comment:
Do we want to introduce the same changes to signing stage?

Reply:
Not sure if I follow the question but... announce phase is used only during key-generation. Readiness signaling is used both for key generation and signing. The problem with mixed cases occurred only for announcement protocol where member addresses were passed from the chain. For readiness, it is not a problem because we are reading addresses from locally stored information using the same code.